queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name::text LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
- [ ] It would be better if we list the `name` of all animals that are neutered and have less than 3 escape attempts. Currently, we are listing all the columns. Let's list only the name as we have in the guidelines
![Screenshot from 2023-08-21 13-58-23](https://github.com/JOdiemo/database-lessons/assets/46388113/b26ca76d-1471-4e2e-94c5-373dda85b687)